### DISCUSSION TITLE
滴答清单体验 - 1nan

### DISCUSSION THREAD

**Discussion by giscus** [yinan-c/pages-comment#15] [Announcements]
# 滴答清单体验 - 1nan

https://yinan.me/ticktick.html

**Comment by versun** [comment]
哈哈，欢迎加入滴答清单，我目前是all in one，除了数据不能完美导出以外，其它都非常满意

**Reply by yinan-c** [reply]
嗯，现在体验下来真的很不错，希望后面能越来越好吧
